src/main/scala/microsites/util/MicrositeHelper.scala

```diff
@@ -229,13 +254,26 @@ class MicrositeHelper(config: MicrositeSettings) {
 
     createFile(sourceFile)
 
-    (faviconFilenames zip faviconSizes)
-      .map { case (name, size) =>
-        (new File(s"$targetDir$jekyllDir/img/$name"), size)
-      }
-      .map { case (file, (width, height)) =>
-        Image.fromFile(sourceFile.toFile).scaleTo(width, height).output(file)
-      }
+    //This is a dirty classloader hack to allow the latest version of Scrimage to work.
+    //This plugin's default classloader is limited and cannot load ImageReader instances.
+    //We get a different ClassLoader that will work, replace it, and then set it back after we're done.
+    //Will be fixed in Scrimage 4.1.0, see: https://github.com/sksamuel/scrimage/issues/217
```

Comment:
maybe log an issue for when we upgrade to 4.1.0?

Reply:
Logged as #535